x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/integrations/callouts/agentless_available_callout.tsx

```diff
@@ -5,19 +5,32 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useCallback } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { EuiIcon, useEuiTheme } from '@elastic/eui';
 import { css } from '@emotion/react';
 
 import { useKibana } from '../../../../../../common/lib/kibana';
 import { LinkAnchor } from '../../../../../../common/components/links';
+import { trackOnboardingLinkClick } from '../../../../../common/lib/telemetry';
 import { CardCallOut } from '../../common/card_callout';
+import { TELEMETRY_AGENTLESS_LEARN_MORE } from '../constants';
 
 export const AgentlessAvailableCallout = React.memo(() => {
   const { euiTheme } = useEuiTheme();
   const { docLinks } = useKibana().services;
 
+  const onClick = useCallback(
+    (e: React.MouseEvent<HTMLAnchorElement>) => {
+      e.preventDefault();
+      trackOnboardingLinkClick(TELEMETRY_AGENTLESS_LEARN_MORE);
+      /* @ts-expect-error: add the blog link to `packages/kbn-doc-links/src/get_doc_links.ts` when it is ready and remove this exit condition*/
+      window.open(docLinks.links.fleet.agentlessBlog, '_blank');
+    },
+    /* @ts-expect-error */
+    [docLinks.links.fleet.agentlessBlog]
+  );
```

Comment:
I'm not sure it's necessary to prevent the default behavior and call the `window.open` manually. Can't we let the LinkAnchor do the navigation work, and we only call `trackOnboardingLinkClick` inside the `onClick`?